package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "strider-gitlab",
-  "version": "1.1.3",
+  "version": "1.2.1",
```

Comment:
Could you remove this, I try to separate the publishing commits from the actual changes, incase multiple changes have to be made.